rotkehlchen/data/import_utils.py

```diff
@@ -0,0 +1,283 @@
+import logging
+from collections import Counter, defaultdict
+from itertools import groupby
+from typing import Any, Dict, List, NamedTuple, Optional
+
+from rotkehlchen.accounting.ledger_actions import LedgerAction, LedgerActionType
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.constants import ZERO
+from rotkehlchen.constants.assets import A_USD
+from rotkehlchen.db.dbhandler import DBHandler
+from rotkehlchen.db.ledger_actions import DBLedgerActions
+from rotkehlchen.errors import NoPriceForGivenTimestamp
+from rotkehlchen.exchanges.data_structures import AssetMovement, Trade
+from rotkehlchen.fval import FVal
+from rotkehlchen.history.price import PriceHistorian
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import (
+    AssetAmount,
+    AssetMovementCategory,
+    Fee,
+    Location,
+    Price,
+    Timestamp,
+    TradeType,
+)
+
+BinanceCsvRow = Dict[str, Any]
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class BinanceEntry(NamedTuple):
+    db: DBHandler
+
+
+class BinanceSingleEntry(BinanceEntry):
+    available_operations: List[str]
```

Comment:
It would be nice to have docstrings for each class here saying what is used for

Reply:
Yep!